js/button/shared/Button.svelte

```diff
@@ -32,7 +33,7 @@
 		id={elem_id}
 	>
 		{#if icon}
-			<img class="button-icon" src={icon.url} alt={`${value} icon`} />
+			<Image class="button-icon" src={icon.url} alt={`${value} icon`} />
```

Comment:
What was wrong with using the <img >element?

Reply:
In the Lite case, `<img>` can't show an image served from the Gradio server because the server tries to serve it via HTTP but it's not available on Lite.
`<Image>` is a wrapper of `<img>` that works on Lite.